### PR TITLE
Update net settings, disable vm, add noatime

### PR DESCRIPTION
**Pull request by pld**
- add more net settings tweaks, these don't look particularly controversial
- disable swap
- set noatime on root partition

- add default to control swap, noatime

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+linux_config_swap_off: True
+linux_config_noatime: False
```
Comment (pld) [inline defaults/main.yml:2]: this seemed dangerous to default True

File: tasks/main.yml [hunk 2]
```diff
@@ -31,4 +33,24 @@
...
+    insertafter: EOF
+    create: True
+  with_items:
+    - "vm.swappiness = 0"
+  when: linux_config_swap_off
+
+- name: Mount up device by UUID
+  mount:
+    path: /
+    src: LABEL=cloudimg-rootfs
```
Comment (pld) [inline tasks/main.yml:52]: This is set to match AWS

File: tasks/main.yml [hunk 3]
```diff
@@ -31,4 +33,24 @@
     - "* hard nofile 65535"
     - "* soft nofile 65535"
     - "* hard nproc 16384"
-    - "* hard nproc 16384"
+    - "* soft nproc 16384"
```
Comment (pld) [inline tasks/main.yml:36]: I think this was a typo

File: tasks/main.yml [hunk 4]
```diff
@@ -9,15 +9,17 @@
...
     - { name: "net.ipv4.tcp_wmem", value: "4096 16384 16777216" }
+    - { name: "net.ipv4.tcp_fin_timeout", value: "15" }
 
 - name: Update sysctl net settings
   sysctl:
     name: "{{ item.name }}"
     value: "{{ item.value }}"
   with_items:
-    - { name: "net.ipv4.tcp_tw_recycle", value: 1 }
+    - { name: "net.ipv4.tcp_tw_reuse", value: 1 }
```
Comment (pld) [inline tasks/main.yml:22]: this is safer

**Review by jasonrogena — APPROVED** [review]

**Pull request merged**